src/main/java/com/google/api/codegen/config/GapicInterfaceConfig.java

```diff
@@ -86,6 +86,11 @@ public String getName() {
     return hasInterfaceNameOverride() ? getInterfaceNameOverride() : getInterface().getSimpleName();
   }
 
+  @Override
+  public String getRawName() {
+    return getInterface().getSimpleName();
+  }
+
```

Comment:
This is only used in one place. Can we just make the calling site do `getInterface().getSimpleName()`?